projects/web-app/vue.config.js

```diff
@@ -11,7 +11,14 @@ module.exports = {
       .loader('vue-loader-v16')
       .end()
       .use('vue-svg-loader')
-      .loader('vue-svg-loader');
+      .loader('vue-svg-loader')
+      .options({
```

Comment:
This did work after we discussed other options? 🥇

Reply:
I found it in an obscure section of the readme, which wasn't necessarily about svgo options